spells/Increasedlogintime.js

```diff
@@ -0,0 +1,51 @@
+function IncreasedlogintimeSpell(context, payload) {
+	// etat du jeu
+    const game = context.getGameState();
+	
+    // si on est pas en partie
+    if (game === null) {
+      context.send('spell:station:increasedlogintime:fail', { errorCode: 0 });
+      return;
+    }
+  
+    // test de la phase de jeu
+    if (game.properties.phase.type !== 2) {
+      context.send('spell:station:increasedlogintime:fail', { errorCode: 1 });
+      return;
+    }
+  
+    // on recois l'index de la station selectionné
+    const stationIndex = payload.station;
+    // on trouve la station correspondante
+    const station = game.stations[stationIndex];
+  
+    // on verifie si la station existe
+    if (station === undefined) {
+      context.send('spell:station:increasedlogintime:fail', { errorCode: 2 });
+      return;
+    }
+  
+    // on change la valeur du login time, ici on est passé de 5 a 15 seconds
+    station.properties.loginTime = 15;
```

Comment:
Change the property `loginMultiplier` instead, and make it `1.5` (https://github.com/VilmaNilsson/VIP2021-game/wiki/Game-state#station).